### ISSUE TITLE
CustomProfileEffectsForMobile

### ISSUE THREAD

**Issue by Ivan951236** [vendetta-mod/ideas#193]
### Discord username

@ivan951236

### Plugin description

CustomProfileEffectsForMobile is a desktop to mobile feature that brings Custom Profile Effects from PC to mobile quickly before discord actually releases it

### Elaborate

Features Included:

- Custom Profile Effects

- Presets (for lazy people)

- Can Be PNG, APNG and GIF (optional if you want a plugin to convert APNGs)

Notes:
Make it so that only person with the same plugin sees it

### ​

- [X] I have verified that this request is indeed original

**Comment by nexpid** [comment]
grabbing this however it's gonna work the same way as Free Profile Colors does (3y3 text in bio)

**Comment by jackmthws** [comment]
Eventually will be a Decor feature

**Comment by nexpid** [comment]
???

**Comment by Ivan951236** [comment]
> grabbing this however it's gonna work the same way as Free Profile Colors does (3y3 text in bio)

I don't think you understand by what i mean Custom Profile Effects, here what i mean
Custom Profile Effects are in a discord shop which is only in desktop and custom profile effects examples is Discord-Os where if you on PC open someone's discord profile then you will be greeted by a profile effect which is a user's choice, it may be bats, it may be Discord-Os anything

**Comment by Ivan951236** [comment]
here is a example to a profile effects on Discord Shop 
![image](https://github.com/vendetta-mod/ideas/assets/101629412/dc56bc3e-1bed-4dc7-833c-db587f31f4c5)

**Comment by janisslsm** [comment]
> I don't think you understand by what i mean Custom Profile Effects, here what i mean Custom Profile Effects are in a discord shop which is only in desktop and custom profile effects examples is Discord-Os where if you on PC open someone's discord profile then you will be greeted by a profile effect which is a user's choice, it may be bats, it may be Discord-Os anything

So you want the desktop feature (store etc) ported to mobile instead of a alternative/recreation

**Comment by jackmthws** [comment]
From the sounds of things they want CUSTOM (as in user created) effects

**Comment by Ivan951236** [comment]
> > I don't think you understand by what i mean Custom Profile Effects, here what i mean Custom Profile Effects are in a discord shop which is only in desktop and custom profile effects examples is Discord-Os where if you on PC open someone's discord profile then you will be greeted by a profile effect which is a user's choice, it may be bats, it may be Discord-Os anything
> 
> So you want the desktop feature (store etc) ported to mobile instead of a alternative/recreation

Well kinda but i meant a custom profile effect of the example, however i don't force you to port it, and i don't force you to recreate either, just recreate if you can, otherwise port it ( :) )

Russian Ivan

**Comment by Yeetov** [comment]
Maybe a port of the Vencord plugin for Vendetta:
![image](https://github.com/vendetta-mod/ideas/assets/69159547/e3690e86-81cb-4a8c-ad3b-6fcd2f8c5650)
https://github.com/sh3devil/faeffect/tree/main

**Comment by Ivan951236** [comment]
> Maybe a port of the Vencord plugin for Vendetta: ![image](https://user-images.githubusercontent.com/69159547/280522845-e3690e86-81cb-4a8c-ad3b-6fcd2f8c5650.png) https://github.com/sh3devil/faeffect/tree/main

No, do not put in a upcoming plugin all the discord's official profile effects just like how the decor plugin disallows you to use discord's official avatar decorations so that the plugin right here is actually not considered both free and custom profile effects

**Comment by Ivan951236** [comment]
also didn't notice some garmmar issue (kinda because i wanted to use and instead of or in a first post which is the time where nexpid didn't notice the plugin request) so i decied to do to fix it :)
